features/recipes_belong_to_users.feature

```diff
@@ -0,0 +1,21 @@
+Feature:
+  As a user
+  In order to protect recipes from editing by others
+  My recipes should belong to me
+
+  Background:
+    Given We have the following user:
+      | email           | password    |
+      | maran@test.com  | 12345678    |
+    And We have the following recipes:
+      | title       	| description                 | ingredients        | directions              |
+      | Meatball-Stew | This will make you sick     | Meat, onion, stuff | Stir it, mix it, eat it |
+
+  Scenario: Visitor can see authors email on index page
+    Given I visit the site
+    Then I should see "maran@test.com"
```

Comment:
Perhaps look for "Posted by: maran@test.com"?
Also, I would add more than 1 recipe in the background step and check for the creator on each div. I can show you how it's done

Reply:
Done